stories/DropZone.stories.tsx

```diff
@@ -7,25 +7,20 @@ export default {
   title: 'Layout/DropZone',
   args: {
     color: 'black',
-    width: '50%',
+    children: <div style={{ backgroundColor: 'blue' }}>hello world</div>,
```

Comment:
Please move the children to the story itself

Reply:
no need to the children now

Reply:
I'd like to see what is rendered inside as children in the story

![image](https://user-images.githubusercontent.com/4118690/146544431-ffdffa79-d6d4-4278-bfa2-6411667d6604.png)

Reply:
children arg added in the active story
but the display of Jsx element is not easily readable , is there any way to improve the children arg display in this story?